seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/executor/InsertOrUpdateBatchStatementExecutor.java

```diff
@@ -121,6 +123,8 @@ public void closeStatements() throws SQLException {
             if (!submitted) {
                 executeBatch();
             }
+        } catch (JdbcConnectorException e) {
```

Comment:
Maybe we should catch `JdbcConnectorException` in https://github.com/apache/seatunnel/blob/f022e409ca1f19ec2bd43ed9a3b2a2154e7e0d47/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/JdbcOutputFormat.java#L205-L213. To assert the `JdbcConnectorErrorCode` to know we can ignore it or not.